runner.py

```diff
@@ -311,7 +331,7 @@ def get_main_parser():
     processor_instance = workflows[args.workflow](
         args.year,
         args.campaign,
-        args.output.replace(".coffea", "").replace("hists_", ""),
+        basename,
```

Comment:
should replace with `outdir` name(this is use to propagate the naming inside the workflows)

Reply:
So this should be `histoutdir`.

Reply:
should be `outdir` we want the name be sync with the  arrays one